apps/www/.eslintrc.cjs

```diff
@@ -0,0 +1,84 @@
+/**
```

Comment:
🤔 @crherman7 what do you think about using the liner from shared package?
and we can extend our linter with a new file for web

for current config I see some errors
✖ 23 problems (23 errors, 0 warnings)

and we can make same updates for `tsconfig.json`

Reply:
I think it would be good to extend upon a shared package for eslint and tsconfig likely

Reply:
let's do this in another PR